noteModel.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Schema.Types.ObjectId;
+
+const NoteSchema = new mongoose.Schema({
+    _id: {
```

Comment:
The `_id` field is added for you automatically by Mongo, so you can leave it out of your schema. I'm honestly unsure what will happen if you leave it in -- it may work, it may not.